soda/src/main/java/it/okkam/opendata/models/soda/SodaResult.java

```diff
@@ -55,4 +56,13 @@ public SodaOwner getOwner() {
   public void setOwner(SodaOwner owner) {
     this.owner = owner;
   }
+
+  public String getPreview_image_url() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00100)